script.js

```diff
@@ -0,0 +1,163 @@
+// Brand Currency form -------------
+let brandObjectHistory = [];
+
+document
+  .getElementById("brand-currency-form")
+  .addEventListener("submit", brandCurrency);
+
+function brandCurrency(event) {
+  event.preventDefault();
+
+  let baseCurrency = document.getElementById("base-currency").value;
+  let eurRate = document.getElementById("eur").value;
+  let usdRate = document.getElementById("usd").value;
+  let gpyRate = document.getElementById("gpy").value;
+
+  const brandObject = {
+    timestamp: Date.now(),
+    base: baseCurrency,
+    date: new Date().toLocaleDateString(),
+    rate: {
+      eurRate: +eurRate,
+      usdRate: +usdRate,
+      gpyRate: +gpyRate,
+    },
+  };
+  brandObjectHistory.push(brandObject);
+
+  return brandObject;
+}
+console.log(brandObjectHistory);
+
+// Finding the most moving value -------------
+document
+  .getElementById("finding-the-most-moving-valuta")
+  .addEventListener("click", showTheMostMovingValuta);
+let mostMovingValutaCount;
+
+function showTheMostMovingValuta(event) {
+  event.preventDefault();
+
+  let eurRateMoving = 0;
+  let usdRateMoving = 0;
+  let gpyRateMoving = 0;
+  let ValutaCountsArray = [];
+
+  for (let j = 0; j < brandObjectHistory.length; j++) {
+    if (brandObjectHistory[j].base === "eur") {
+      eurRateMoving++;
+    } else if (brandObjectHistory[j].base === "usd") {
+      usdRateMoving++;
+    } else if (brandObjectHistory[j].base === "gpy") {
+      gpyRateMoving++;
+    }
+  }
```

Comment:
@katayouny Oh I understand now, but I think you are ahead of time :D Although it was description od the task I think you should focus on these tasks for now 

```
 Create a form to insert a brand new currency rate from a to b.
 Create a form to covert an amount of money from x currency to y based on the rates provided.
 optional bonus: create a form to update existing currency conversion with a new rate.
```
 The rest will be done during other weeks :D